contrib/olivere/elastic/elastictrace.go

```diff
@@ -29,6 +29,8 @@ const componentName = "olivere/elastic"
 
 func init() {
 	telemetry.LoadIntegration(componentName)
+	tracer.MarkIntegrationImported("gopkg.in/olivere/elastic.v5")
```

Comment:
We mark both here? 🤔

Reply:
😔 yeah, for some reason this contrib package supports both v3 and v5, so I think marking both is correct...?